exonum/src/messages/signed.rs

```diff
@@ -289,4 +311,26 @@ mod tests {
         let msg2 = Verified::<AnyTx>::from_bytes(bytes.into()).unwrap();
         assert_eq!(msg, msg2);
     }
+
+    #[test]
```

Comment:
I've meant also failure tests (e.g., trying to convert `Verified<AnyTx>` from `Verified<Precommit>` serialization, or from an overly short byte sequence), but, as I've said, this can be done separately. (Please add a ticket in Jira in this case.)

Reply:
Created ticket: [ECR-4073]